pkg/util/iptables/iptables.go

```diff
@@ -49,6 +51,18 @@ type Interface interface {
 	DeleteRule(table Table, chain Chain, args ...string) error
 	// IsIpv6 returns true if this is managing ipv6 tables
 	IsIpv6() bool
+	// Save calls `iptables-save` for table.
+	Save(table Table) ([]byte, error)
```

Comment:
argument to createProxier:
*fakeIptables does not implement iptables.Interface (missing Restore method)
_output/local/go/src/k8s.io/kubernetes/pkg/proxy/proxier_test.go:508: cannot use fakeIptables literal (type *fakeIptables) as type iptables.Interface in argument to createProxier:
*fakeIptables does not implement iptables.Interface (missing Restore method)
_output/local/go/src/k8s.io/kubernetes/pkg/proxy/proxier_test.go:560: cannot use fakeIptables literal (type *fakeIptables) as type iptables.Interface in argument to createProxier:
*fakeIptables does not implement iptables.Interface (missing Restore method)
_output/local/go/src/k8s.io/kubernetes/pkg/proxy/proxier_test.go:560: too many errors
FAIL	k8s.io/kubernetes/pkg/proxy [build failed]